src/test/kotlin/pt/up/fe/ni/website/backend/BackendApplicationTests.kt

```diff
@@ -6,5 +6,6 @@ import org.springframework.boot.test.context.SpringBootTest
 @SpringBootTest
 class BackendApplicationTests {
     @Test
-    fun contextLoads() {}
+    fun contextLoads() {
+    }
```

Comment:
Was this a change from the linter? Looks weirder to me but if it is then just leave it

Reply:
It was, found it weird too.

Reply:
Changed it, I changed the link to href because I didn't really understand if I was supposed to delete it, either way, I can delete it quickly if so.